src/Fantomas/CodePrinter.fs

```diff
@@ -574,7 +574,14 @@ and genExpr astContext synExpr =
         !- s +> sepSpace +> sepOpenS +> genExpr { astContext with IsNakedRange = true } e 
         +> ifElse (checkBreakForExpr e) (sepNln +> sepCloseSFixed) sepCloseS
     // This supposes to be an infix function, but for some reason it isn't picked up by InfixApps
-    | App(Var "?", e::es) -> genExpr astContext e -- "?" +> col sepSpace es (genExpr astContext)
+    | App(Var "?", e::es) ->
+        match es with
+        | SynExpr.Ident(_)::_
```

Comment:
We should also use parens in this case - this means user used `x?(k)` and we shouldn't  remove parens in that case, see https://github.com/fsprojects/fantomas/issues/369#issue-384812957